src/ontology/edits/oeo-physical.omn

```diff
@@ -4648,7 +4655,8 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/391",
         rdfs:label "declared net capacity"@en
     
     SubClassOf: 
-        OEO_00000350
+        OEO_00000350,
+        OEO_00000031 or (OEO_00020056 some OEO_00000334)
```

Comment:
this should be `OEO_00020056 some (OEO_00000031 or OEO_00000334)`

Reply:
True, thanks.